IM_calculation/IM/im_calculation.py

```diff
@@ -306,6 +309,37 @@ def calculate_pSAs(
                 ][i]
 
 
+def calculate_IESDR(
+    DT,
+    accelerations,
+    im_options,
+    result,
+    station_name,
+    waveform_acc,
+    im,
+    comps_to_store,
+    comps_to_calculate,
+):
+    # for the time being, following 3 parameters are fixed
+    z = 0.05  # damping ratio
+    alpha = 0.05  # strain hardening ratios
+    dy = 0.025  # strain hardening ratios
```

Comment:
These should be function parameters with the given values as defaults, so that we can use different values later on if needed

Reply:
Good point. Will update shortly